src/components/register/register.jsx

```diff
@@ -0,0 +1,172 @@
+/* eslint-disable react/no-children-prop */
+import React, { useState } from 'react';
+import { instanceOf, string } from 'prop-types';
+import { useNavigate } from 'react-router-dom';
+import {
+  FormControl,
+  FormLabel,
+  InputGroup,
+  Input,
+  Heading,
+  InputRightAddon,
+  Button,
+  Flex,
+  VStack,
+  Center,
+  Text,
+} from '@chakra-ui/react';
+import { Cookies, withCookies } from '../../utils/cookie_utils';
+import { registerWithEmailAndPassword } from '../../utils/auth_utils';
+
+const Register = ({ cookies, inviteFirstName, inviteLastName, inviteEmail, inviteRole }) => {
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'cookies' is defined but never used.